.drone.yml

```diff
@@ -2,6 +2,10 @@ workspace:
   base: /go
 
 pipeline:
+  clone:
+    image: plugins/git
+    tags: true
```

Comment:
But surely that's a change you want on your drone file, not in this one... The tag that you just removed is a docker image tag, not a git tag